student/src/main/java/tds/student/services/remote/RemoteOpportunityService.java

```diff
@@ -0,0 +1,141 @@
+package tds.student.services.remote;
+
+import TDS.Shared.Browser.BrowserInfo;
+import TDS.Shared.Exceptions.ReturnStatusException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+import tds.common.Response;
+import tds.common.ValidationError;
+import tds.exam.Exam;
+import tds.exam.OpenExamRequest;
+import tds.student.services.abstractions.IOpportunityService;
+import tds.student.services.data.ApprovalInfo;
+import tds.student.sql.abstractions.ExamRepository;
+import tds.student.sql.data.OpportunityInfo;
+import tds.student.sql.data.OpportunityInstance;
+import tds.student.sql.data.OpportunitySegment;
+import tds.student.sql.data.OpportunityStatus;
+import tds.student.sql.data.OpportunityStatusChange;
+import tds.student.sql.data.OpportunityStatusExtensions;
+import tds.student.sql.data.TestConfig;
+import tds.student.sql.data.TestSegment;
+import tds.student.sql.data.TestSelection;
+import tds.student.sql.data.TestSession;
+import tds.student.sql.data.Testee;
+
+@Service("integrationOpportunityService")
+@Scope("prototype")
```

Comment:
Also, properties only seem to work at the service layer and down.  The Web layer doesn't load the progman properties. :/